src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -61,21 +74,54 @@ const RunTranscriptionView = ({ closeModal, nextView }: Props) => {
     ) {
       return;
     }
+    const { mediaFilePath } = currentProject;
+
+    if (mediaFilePath === null) {
+      return;
+    }
+    const setCurrentProject = (project: RuntimeProject) => {
+      return dispatch(projectCreated(project));
+    };
+
+    const extractProjectAudio = async () => {
+      const audioFilePath = await extractAudio(currentProject);
+      const projectWithAudioExtract = await updateProjectWithExtractedAudio(
+        currentProject,
+        audioFilePath
+      );
+      if (projectWithAudioExtract === null) {
+        return;
+      }
+      setCurrentProject(projectWithAudioExtract);
+    };
 
     setAsyncState(AsyncState.LOADING);
 
-    requestTranscription(currentProject)
-      .then((transcription) => {
-        if (transcription === null) {
+    const startProcessing = async () => {
+      await updateProjectWithMedia(currentProject, mediaFilePath);
+      await extractProjectAudio();
+      await processTranscription(currentProject)
+        .then((transcription) => {
```

Comment:
The idea with await is that the contents of the `then` can be extracted, i.e.

```
const transcription = await processTranscription(currentProject);
... then do things with transcription
```

This should be in a try/catch block as well so that we can set the async state to ERROR if it fails